app/controllers/admin/routes.py

```diff
@@ -74,10 +72,19 @@ def createEvent(templateid, programid=None):
     if request.method == "POST":
         eventData.update(request.form.copy())
 
-    if program:
-        # TODO need to handle the multiple programs case
+    if program and request.method == "GET":
         eventData["program"] = program
 
+        if program.contactName and program.contactEmail:
+            # TODO need to handle the multiple programs case
+            eventData['contactName'] = program.contactName
+            eventData['contactEmail'] = program.contactEmail
+
+        else:
+            # sets contact name and email to CELTS director (admin) if there is none set
+            eventData['contactName'] = "CELTS"
+            eventData['contactEmail'] = app.config['celts_admin_contact']
```

Comment:
We want this general contact to be available for all events, not just ones with a program, so it needs to happen outside of the program check. Also, let's make the name "CELTS Admin"

Reply:
resolved